examples/jsm/nodes/display/ViewportNode.js

```diff
@@ -102,9 +99,9 @@ class ViewportNode extends Node {
 
 				// follow webgpu standards
 
-				const resolution = viewportResolution.build( builder );
+				const resolution = builder.getNodeProperties( viewportResolution ).outputNode.build( builder );
```

Comment:
Why does `viewportResolution.build( builder )` not work? (Possibly it could be fixed with one of the changes from my PR -- introducing `Node.fullBuild()`, which runs all builds from all build stages on a node)

Reply:
/ping @sunag